src/locales/ar.json

```diff
@@ -266,7 +267,7 @@
         "role": "مطورة ويب"
       },
       "tarkan": {
-        "name": "طاركان منصوراوغلو",
+        "name": "تاركان منصوراوغلو",
```

Comment:
wait do I have an Arabic name?

Reply:
I just wrote it in the Arabic alphabet for pronounciation :joy: